examples/employee-app/src/main/resources/META-INF/native-image/helidon-example-reflection-config.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
You don't need this file unless you intend to support native image with GraalVM.

Reply:
Deleted